dataprofiler/profilers/float_column_profile.py

```diff
@@ -57,7 +78,39 @@ def __add__(self, other):
                                  other.__calculations)
 
         if "precision" in merged_profile.__calculations:
-            merged_profile.precision = min(self.precision, other.precision)
+
+            if self.precision['min'] is None:
+                merged_profile.precision = copy.deepcopy(other.precision)
+            elif other.precision['min'] is None:
+                merged_profile.precision = copy.deepcopy(self.precision)                
+            else:
+                merged_profile.precision['min'] = min(
+                    self.precision['min'], other.precision['min'])
+                merged_profile.precision['max'] = max(
+                    self.precision['max'], other.precision['max'])
+                merged_profile.precision['sum'] = \
```

Comment:
do we really need sum?

Reply:
Although it isn't a useful metric it has to get stored somewhere for the sake of calculating mean.

Reply:
Yes! Wont need it returned, but need it to calculated mean & variance between chunks.